api/tests/test_asset_model_number_api.py

```diff
@@ -0,0 +1,180 @@
+from django.test import TestCase
+from unittest.mock import patch
+from rest_framework.test import APIClient
+from rest_framework.reverse import reverse
+
+from core.models import User, AssetCategory, AssetSubCategory, \
+    AssetType, AssetMake, AssetModelNumber
+
+client = APIClient()
+
+
+class AssetModelNumberAPITest(TestCase):
+    """ Tests for the Asset Model Number endpoint"""
+
+    def setUp(self):
+        self.user = User.objects.create(
+            email='testuser@gmail.com', cohort=19,
+            slack_handle='tester', password='qwerty123'
+        )
+
+        self.asset_category = AssetCategory.objects.create(
+            category_name="Accessories"
+        )
+        self.asset_sub_category = AssetSubCategory.objects.create(
+            sub_category_name="Key Board",
+            asset_category=self.asset_category
+        )
+        self.asset_type = AssetType.objects.create(
+            asset_type="Dell",
+            asset_sub_category=self.asset_sub_category
+        )
+
+        self.asset_label = AssetMake.objects.create(
+            make_label="ASSET-LABEL",
+            asset_type=self.asset_type
+        )
+
+        self.asset_model_no = AssetModelNumber.objects.create(
+            model_number="TEST-MODEL-NO",
+            make_label=self.asset_label
+        )
+
+        self.asset_model_no_url = reverse('asset-models-list')
+        self.token_user = 'testtoken'
+
+    def test_non_authenticated_user_get_asset_model_number(self):
+        response = client.get(self.asset_model_no_url)
+        self.assertEqual(response.data, {
+            'detail': 'Authentication credentials were not provided.'
+        })
+
+    @patch('api.authentication.auth.verify_id_token')
+    def test_can_post_asset_model_number(self, mock_verify_token):
```

Comment:
The strings in this function use both single and double quotes. Prefer single quotes since they are more commonly used in the project.